packages/cli/api-importers/v2-importer-commons/src/converters/ExampleConverter.ts

```diff
@@ -540,19 +534,67 @@ export class ExampleConverter extends AbstractConverter<AbstractConverterContext
             })
         );
 
-        const isValid = resultsByKey.every((entry) => entry.result.isValid);
+        const allOfResults = await Promise.all(
+            (resolvedSchema.allOf ?? []).map(async (subSchema, index) => {
+                const exampleConverter = new ExampleConverter({
+                    breadcrumbs: [...this.breadcrumbs, `allOf[${index}]`],
+                    context: this.context,
+                    schema: { ...resolvedSchema, ...subSchema, allOf: undefined },
+                    example: this.example,
+                    depth: this.depth + 1,
+                    generateOptionalProperties: this.generateOptionalProperties
+                });
+                return await exampleConverter.convert();
+            })
+        );
+
+        const isValid =
+            resultsByKey.every((entry) => entry.result.isValid) && allOfResults.every((result) => result.isValid);
```

Comment:
what happens when one entry is invalid? (wouldn't that generate an invalid example then?)